app/src/main/java/com/example/soptexampleproject/presentation/home/screens/fragment/profilechildfragment/FollowerAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.soptexampleproject.presentation.home.screens.fragment.profilechildfragment
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.soptexampleproject.data.model.UserData
+import com.example.soptexampleproject.databinding.ItemSampleListBinding
+
+class FollowerAdapter:RecyclerView.Adapter<FollowerAdapter.MyViewHolder>() {
+
+    private val _userList= mutableListOf<UserData>()
+
+     val userList get() = _userList!!
```

Comment:
이것도 그냥 못넘어가겠네요 ㅎㅎ 
이 놈의 Reformat Code,,,

```suggestion
    private val _userList= mutableListOf<UserData>()

    val userList get() = _userList!!
```

Reply:
완벽했다고 생각했지만 아니었네요 ㅎ